README.md

```diff
@@ -1,2 +1,31 @@
 # wl-tracking
-This package allows you to add basic tracking capabilities to your applications
+
+This monorepos contains some packages related to tracking.
+
+
+# `@workleap/tracking`
+
+This package allows you to add basic tracking capabilities to your applications. It exposes a single `track` function that calls a tracking endpoint that conforms to the Workleap Platform Tracking API in the environnement URL you have provided.
+
+To use it you must first call the `buildTrackingFunction` factory, which will return you a tracking function that you can then use in your code
```

Comment:
We can keep "buid" for backward compatility reasons, but that's an "odd" choice. I don't think this is a "builder" pattern here.

I would probably go for "create".